newswires/app/controllers/ViteController.scala

```diff
@@ -61,12 +65,15 @@ class ViteController(
         .replaceAll("@csrf\\.value", csrf.value)
     }
 
-    def injectClientConfig(body: String): String = {
+    def injectClientConfig[A](
+        body: String
+    )(implicit request: UserRequest[A]): String = {
```

Comment:
No need for the following implicit param `implicit request: UserRequest[A]` because it's defined on the parent function.